backend/apps/volontulo/tests/views/api/test_password_change.py

```diff
@@ -0,0 +1,107 @@
+# pylint: disable=missing-docstring
+from unittest.mock import patch
+
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APITestCase
+
+from apps.volontulo.factories import UserFactory
+
+ENDPOINT_URL = reverse('password_change')
+
+
+class TestPasswordChange(APITestCase, TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.password = 'admin12345'
+        cls.user = UserFactory.create()  # type: User
+        cls.user.set_password(cls.password)
+        cls.user.save()
+
+    @classmethod
+    def tearDownClass(cls):
```

Comment:
It shouldn't be necessary - all tests should be encapsulated in database transaction.